configs/cameroon.js

```diff
@@ -670,6 +670,144 @@ module.exports = {
         }
       ]
     },
+    GROUP_TEST: {
+      grouptype: 'default',
+      order: 1,
+      label: {
+        en: 'TEST',
+        fr: 'TEST',
+        es: 'TEST',
+        pt: 'TEST',
+        id: 'TEST',
+        zh: 'TEST',
+        ka: 'TEST'
```

Comment:
@vaidotasp I'm thinking we may need a clarifying comment that explains that `GROUP_TEST` was integrated to account for more layer types?

Reply:
GROUP_TEST exists in development environment only, this cameroon.js file is not something that will be ever deployed. But I can put some comments if that's helpful!

Reply:
Yeah I was thinking comments could help since `GROUP_TEST` has layer info for India and the config file is largely for cameroon, but I'm flexible either way